views/items.py

```diff
@@ -92,6 +94,9 @@ def patch(self, item_uuid):
         if availability:
             obj.availability = availability
 
+        if category:
+            obj.category = category
```

Comment:
Come detto a voce verifichiamo se, nel caso in cui il category sia mancante o vuoto, la validazione fallisce. In questo caso, dovremmo poter eliminare `if category` in quanto a questo punto `category` sarà sempre presente.